velox/functions/prestosql/Arithmetic.h

```diff
@@ -646,3 +646,10 @@ struct CosineSimilarityFunction {
 
 } // namespace
 } // namespace facebook::velox::functions
+
+template <>
+struct fmt::formatter<std::errc> : formatter<int> {
+  auto format(std::errc s, format_context& ctx) {
+    return formatter<int>::format(static_cast<int>(s), ctx);
+  }
+};
```

Comment:
Based on the CI failures, it looks like we'll need this to also be accessible from `velox/Expression/CastExpr-inl.h`. It's a bit ugly, but `velox/common/base/Exceptions.h` is one possibility.

Seems like there was discussion about adding a formatter for errc directly into fmt, but it didn't go forward so that downstream has control over formatting it. https://github.com/fmtlib/fmt/pull/3574

Reply:
I agree with @andrewseidl. We should probably create a separate header with formatters for missing `std::` UDTs and include that in `Exceptions.h`. Multiple Velox sources could be using them.